src/components/Dashboard/Dashboard.js

```diff
@@ -91,50 +122,67 @@ const Dashboard = ({navigation}) => {
   }
 
   const showPairInfo = (quote) => {
-    navigation.navigate('PairInfoQuotes', {quoteParams: quote, title: quote.symbol})
+    props.navigation.navigate('PairInfoQuotes', { quoteParams: quote, title: quote.symbol })
```

Comment:
send params to navigation creates an unnecessary mess in the app - better to use the store for this